radio/src/translations/ua.h

```diff
@@ -0,0 +1,1257 @@
+#define TR_MULTI_WBUS                  "Вихід"
+#define TR_MULTI_TELEMETRY             "Телеметрія"
+#define TR_MULTI_VIDFREQ               TR("Від.Част.", "Відео частота")
+#define TR_RF_POWER                    "RF Power"	/* use english */
+#define TR_MULTI_FIXEDID               TR("FixedID", "Fixed ID")		/* use english */
+#define TR_MULTI_OPTION                TR("Опція", "Значення опції")
+#define TR_MULTI_AUTOBIND              TR(INDENT "Прив'язка на кан.", INDENT "Прив'язка на канал")
+#define TR_DISABLE_CH_MAP              TR("Нема.кан.карти", "Вимкнути карту каналів")
+#define TR_DISABLE_TELEM               TR("Нема телеметрії", "Вимкнути телеметрію")
+#define TR_MULTI_DSM_AUTODTECT         TR(INDENT "Автовизначення", INDENT "Формат автовизначення")
+#define TR_MULTI_LOWPOWER              TR(INDENT "Низька потужн.", INDENT "Ркжим нізької потужності")
+#define TR_MULTI_LNA_DISABLE           INDENT "Вимкн. LNA"
+#define TR_MODULE_TELEMETRY            TR(INDENT "S.Port", INDENT "S.Port link")		/* use english */
+#define TR_MODULE_TELEM_ON             TR("ON", "Увімкнено")
+#define TR_DISABLE_INTERNAL            TR("Дект. внутр.", "Деактивувати внутр. RF")
+#define TR_MODULE_NO_SERIAL_MODE       TR("Не сер.реж.", "Не в серіальному режимі")
+#define TR_MODULE_NO_INPUT             TR("Немає входу", "Немає серіального входу")
+#define TR_MODULE_NO_TELEMETRY         TR3("Нема телеметрії", "Нема телеметрії", "Телеметрія не визначена")
+#define TR_MODULE_WAITFORBIND          "Прив'язка до завантаж. протоколу"
+#define TR_MODULE_BINDING              TR("Прив'язка...","Прив'язую")
+#define TR_MODULE_UPGRADE_ALERT        TR3("Потрібен апгр.", "Потрібен апгрейд модулю", "Потрібен\nАпгрейд модулю")
+#define TR_MODULE_UPGRADE              TR("Апг.реком.", "Рекомендовано апгрейд модулю")
+#define TR_REBIND                      "Потрібна переприв'язка"
+#define TR_REG_OK                      "Регістрація ok"
+#define TR_BIND_OK                     "Прив'язка успішна"
+#define TR_BINDING_CH1_8_TELEM_ON      "Ch1-8 Телем ON"
+#define TR_BINDING_CH1_8_TELEM_OFF     "Ch1-8 Телем OFF"
+#define TR_BINDING_CH9_16_TELEM_ON     "Ch9-16 Телем ON"
+#define TR_BINDING_CH9_16_TELEM_OFF    "Ch9-16 Телем OFF"
+#define TR_PROTOCOL_INVALID            TR("Прот.невірн.", "Протокол невірний")
+#define TR_MODULE_STATUS               TR(INDENT "Статус", INDENT "Статус модулю")
+#define TR_MODULE_SYNC                 TR(INDENT "Синх", INDENT "Синхро протоколу")
+#define TR_MULTI_SERVOFREQ             TR("Частота серво", "Частота оновлення серво")
+#define TR_MULTI_MAX_THROW             TR("Max. Хід", "Увімкн. max. хід")
+#define TR_MULTI_RFCHAN                TR("RF Канал", "Обери RF канал")
+#define TR_AFHDS3_RX_FREQ              TR("RX част.", "RX частота")
+#define TR_AFHDS3_ONE_TO_ONE_TELEMETRY TR("Unicast/Тел.", "Unicast/Телеметрія")
+#define TR_AFHDS3_ONE_TO_MANY          "Multicast"
+#define TR_AFHDS3_ACTUAL_POWER         TR("Факт.пот.", "Фактична потужність")
```

Comment:
This is more consistent with other places you used this shortened term
```suggestion
#define TR_AFHDS3_ACTUAL_POWER         TR("Факт.потужн.", "Фактична потужність")
```